lib/mollie/client.rb

```diff
@@ -60,5 +60,14 @@ def refund_payment(payment_id)
       JSON.parse(response.body)
     end
 
+    def methods(method = nil)
+      response = self.class.get("/methods/#{method}",
+        :headers => {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.